# Motor feedback data as reported

std_msgs/Header header

float64 torque_output   # Measured torque output of the motor in N-m
float64 torque_input    # Measured torque input of the motor in N-m
